docs/running-workflows/supported-systems/cwl/index.md

```diff
@@ -1,3 +1,193 @@
 # CWL
 
 The Common Workflow Language standard originated in life science community.
```

Comment:
Please expand the introductory text a bit, for example:

```The Common Workflow Language standard for describing computational workflows originated in the life science community. Please see:

- CWL project website: https://www.commonwl.org/
- CWL user guide: https://www.commonwl.org/user_guide/

## Supported versions

The REANA platform implements CWL standard by means of using [cwltool](https://github.com/common-workflow-language/cwltool) reference implementation. The different REANA release series versions support different `cwltool` versions. The following table summarises the supported `cwltool` versions in each REANA release series:

... table ...
```

BTW please also dig out the past releases; it may be good to be complete here.